phpstan.neon

```diff
@@ -46,4 +48,5 @@ parameters:
         -
             message: '/^Parameter #1 \$finder of method PhpCsFixer\\Config::setFinder\(\) expects iterable<string>, int given\.$/'
             path: tests/ConfigTest.php
+        - '/has no return typehint specified\.$/'
```

Comment:
Half of the errors from the baseline are missing return types. A lot of them would be `: void`. On PHP 5.6 this can be fixed by using `@return void` PHPDoc tag instead but our current code style forbids that so let's ignore those for now.

Also, let's drop PHP 5.6 support please.